maquetado/scss/partials/_tech.scss

```diff
@@ -31,6 +29,15 @@ $border-radius: 2%;
     width: 100%;
     z-index: -1;
   }
+
+  &:hover::after {
+    left: 10px;
```

Comment:
I'd recommend not moving elements using `top`/`left`/`bottom`/`right` properties since changing them trigger the [Layout](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-tree-construction) function on the browser's render process. Try to use `translate` instead. ✌️